frodo.js

```diff
@@ -2,40 +2,41 @@
  * Extras Plugin
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!